docs/developer-guide/etendo-copilot/bundles/dev-assistant.md

```diff
@@ -4,17 +4,194 @@ tags:
+
+#### Usage Example
+
+1. To use this assistant, it is necessary to log in as `System Administrator` role and set the role access. For this, go to the **Assistant** window, configure Event Handler Creator and synchronize it. Then, go to the **Assistant Access** window and give access to the role.  
+
+2. Open Copilot and select Event Handler Creator, then ask to the assistant what you need to create.
+
+    ![eventhandler.png](../../../assets/developer-guide/etendo-copilot/bundles/dev-assistant/EH1.png)
+
+3. Then this is the result given by the assistant.
+
+    ![eventhandler.png](../../../assets/developer-guide/etendo-copilot/bundles/dev-assistant/EH2.png)
+
+### Background Process Creator
+
+**Background Process Creator** is an assistant specialized in the automatic creation of Background Processes in Java. The assistant reads previously indexed code and uses examples of classes extending `DalBaseProcess` to build new Background Processes.
+
+#### Tools
+
+- [API Call Tool](../available-tools/openapi-tool.md)
+
+- [Read File Tool](../available-tools/read-file-tool.md)
+  
+- [Write File Tool](../available-tools/write-file-tool.md)
+
+#### Functionality.
+
+Background Process Creator generate Background Processes in Java using indexed examples of classes that extend `DalBaseProcess`. To configure the assistant properly, you need to provide the following parameters:
+
+- **Java package**: The Java package where the new process will be saved. It should follow the format `java.package.of.the.module`.
+- **Name**: The name of the Java file to be created.
+- **Search Key**: A key that will be used to locate the process in other windows when necessary.
+- **Code description**: The purpose and logic that the Background Process should fulfill.
+
+#### Usage Example
+
+1. To use this assistant, it is necessary to log in as `System Administrator` role and set the role access. For this, go to the **Assistant** window, configure Background Process Creator and synchronize it. Then, go to the **Assistant Access** window and give access to the role.
+
+    ![background.png](../../../assets/developer-guide/etendo-copilot/bundles/dev-assistant/BG1.png)
+
+
+2. Open Copilot and select Background Process Creator, then ask to the assistant what you need to create.
+
+3. Then this is the result given by the assistant.
+
+    ![background.png](../../../assets/developer-guide/etendo-copilot/bundles/dev-assistant/BG2.png)
+
+    ![background.png](../../../assets/developer-guide/etendo-copilot/bundles/dev-assistant/BG3.png)
+
+### Webhook Creator
+
+**Webhook Creator** is an assistant designed to facilitate the creation and registration of webhooks in Java. A webhook acts as a communicator between applications or services, allowing data to be automatically sent from one system to another in real-time when a specific event occurs.
+
+#### Tools
+
+- [API Call Tool](../available-tools/openapi-tool.md)
+
+- [Read File Tool](../available-tools/read-file-tool.md)
```

Comment:
It would be good for all assistants to explain what each tool is used for in that specific assistant.